.gitlab/lassen-jobs.yml

```diff
@@ -26,18 +26,18 @@ clang_11_0_0:
 
 gcc_8_3_1:
   variables:
-    SPEC: "%gcc@8.3.1 cxxflags='-finline-functions -finline-limit=20000' cflags='-finline-functions -finline-limit=20000'"
+    SPEC: "+openmp %gcc@8.3.1 cxxflags='-finline-functions -finline-limit=20000' cflags='-finline-functions -finline-limit=20000'"
   extends: .build_and_test_on_lassen
 
 xl_16_1_1_11:
   variables:
-    SPEC: "%xl@16.1.1.11 cxxflags='-qthreaded -std=c++14 -O3 -qxlcompatmacros -qlanglvl=extended0x -qalias=noansi -qhot -qpic -qsmp=omp -qsuppress=1500-029 -qsuppress=1500-036'"
+    SPEC: "+openmp %xl@16.1.1.11 cxxflags='-qthreaded -std=c++14 -O3 -qstrict -qxlcompatmacros -qlanglvl=extended0x -qalias=noansi -qhot -qpic -qsmp=omp -qsuppress=1500-029 -qsuppress=1500-036'"
```

Comment:
Should we document why we're using -qstrict?

Reply:
I thought it would be batter to describe in an issue, so we can try to see if it works at some point.  Although, Mike C. this sort of thing has been an issue with XL for many years.